client/client.go

```diff
@@ -771,6 +771,12 @@ func (c *Client) NodeID() string {
 	return c.config.Node.ID
 }
 
+// NodeIP returns the IP address the node is listening
+func (c *Client) NodeIP() string {
+	addr, _, _ := net.SplitHostPort(c.config.Node.HTTPAddr)
```

Comment:
For clients, the RPC address would be the address seen by the servers that shows up in `node status` outputs and the like. Maybe we should use that instead of the HTTP address, which is only going to be used by external consumers?

Either way, should we call this method and the metrics label for the specific IP (RPC or HTTP) we're using, so that if we come back later to add another IP we can do so?

Reply:
Sounds good. I will use the RPC address and rename the method (`NodeRPCIP`) and metric `node_rpc_ip`. Thanks!